src/components/templates/PosterPage/components/ShareModal/ShareModal.scss

```diff
@@ -0,0 +1,82 @@
+@import "scss/constants.scss";
```

Comment:
As before, most of the px values can be and should be imported from `constants`. If you can't find the exact value, either use the closest one, if the difference is small or create a new constant

Reply:
Looks like there are still a lot of `px` values and similar being used directly here (eg. `height`, `top`, `left`, etc)

@mike-lvov Are you ok with these as they are, or should they be extracted further?

Reply:
@mike-lvov That confuses me, since one of the main things you seem to call out in PR reviews is extracting these sorts of things..?

Reply:
Using the constants everywhere where possible, yes. heights/width, not always(but I don't know if we should do it, if we are not reusing them, honestly). Distances - I don't think I ever called out